src/gui/sdlmain.cpp

```diff
@@ -2698,6 +2695,207 @@ static SDL_Rect calc_viewport_pp(int win_width, int win_height)
 		return calc_viewport_fit(width, height);
 }
 
+void remove_window()
+{
+	if (sdl.window) {
+		SDL_DestroyWindow(sdl.window);
+		sdl.window = nullptr;
+	}
+}
```

Comment:
`remove_window` is only used locally in this cpp file, so should be `static` function.
Might have to move this function up to line ~832 so it's visible before its used.

Reply:
Done.